src/trupti/String/Swap/SwapStringWithoutTempVar.java

```diff
@@ -0,0 +1,25 @@
+package trupti.String.Swap;
+/*program 2: swap 2 String without using temporary variable.
+input : input1 = "Techno" , input2 = "Credits";
+output : input1 = "Credits", input2 = "Techno"
+
+
+Hint : you can use subString method.*/
+
+public class SwapStringWithoutTempVar {
+	void getSwapString() {
+		String input1="Techno";
+		String input2="Credits";
+		input1=input1+input2;
+		input2=input1.substring(0,6 );
```

Comment:
this not be hard coded, please make this dynamic! take of this next time we should not write any hard coded value